source/documentation/standards/code-freezes.html.md.erb

```diff
@@ -0,0 +1,133 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Avoid code freezes
+last_reviewed_on: 2023-06-19
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+Teams building systems for the MOJ should avoid code freezes whenever
+possible as they generally increase the risks to your service and those
+services around it.
+
+While code freezes are intended to reduce uncertainty and change when
+making changes to a system, they are typically a symptom of low trust
+in existing change processes around a given system, or of a previous
+traumatic event in the system's past.
+
+In most cases, it is better to make the system safer to change through
+[smaller, more regular changes](https://www.gov.uk/service-manual/technology/deploying-software-regularly)
+because [regular releases reduce risk](https://gds.blog.gov.uk/2012/11/02/regular-releases-reduce-risk/).
+
+## Code freezes introduce many risks
+
+There are various types of risk that code freezes introduce, both to
+the team's ability to resolve incidents and to the risk and impact
+profile of changes before, during, and after the code freeze.
+
+### Exceptional processes create uncertainty, and risk extending incidents
+
+In a team's day-to-day work, they will use a standard process for
+releasing changes to their systems. If they make changes regularly ([as
+they should](https://www.gov.uk/service-manual/technology/deploying-software-regularly)),
+they will be practiced experts in their normal processes. These
+processes will likely be well documented, well tested, and ideally
+primarily automated.
+
+Even during a code freeze, however, incidents can happen that are out of
+everyone's control, and that cannot be resolved without making changes.
+
+Code freezes introduce a new process with which the team aren't
```

Comment:
```suggestion
Code freezes can introduce a new process with which the team may not be
```

Reply:
If you currently implement code freezes, you'll know the process: so it won't always introduce a new process.